pharmacies/utils/utils.py

```
@@ -170,7 +174,7 @@ def check_if_pharmacy_exists(name: str, phone: str) -> bool:
     return True if pharmacy else False
 
 
-def add_scraped_data_to_db(scraped_data, city_name: str) -> None:
+def add_scraped_data_to_db(scraped_data: list[dict[str, Any]], city_name: str) -> None:
     city: City = City.objects.get(name=city_name)
     if not city:
         raise ValueError("City not found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)